IFC4x3/Properties/n/NominalVoltage_3lq2xqPH9FnvgKOz8DOPYt/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NominalVoltage_3lq2xqPH9FnvgKOz8DOPYt" Name="NominalVoltage" UniqueId="efd02ef4-6512-4fc7-9a94-63d20d6198b7" PrimaryDataType="IfcElectricVoltageMeasure" />
```

Comment:
Think I have seen this before. Is it the same property?

Reply:
This looks like the first one. Could merge to this one when next relevant merge happens